applications/msai.md

```diff
@@ -0,0 +1,290 @@
+
+ID verification and data enhancement could be integrated through a Sidecar and or Pallet approach allowing other app developers within the ecosystem to reuse the chain without needing their own backend. 
+For a simpler initial design we would enhance the off chain worker storage api[6]. Distributed data could be verified as authentic by having a secure hash stored on the chain [4].
+With trust being established by the blockchain layer, any blockchain node that provides faulty data as verified through smart contract hashes will be kicked out and forfiet their staking in the parachains native token.
+We also propose specific data format for anonymous tags such that no user identified private data would be exposed
+during the whole process. 
+The initial user of this parachain would be our own EMR application where users store any medical documents with automated information extraction, along with authenticated GPDR opt out.
+Sample screen of users view of how the data will be presented in our implementation can be seen[here](https://docs.google.com/presentation/d/e/2PACX-1vR5uQkX9I_UdCQwDQpNUkyaIQKtYWfY0_ZhDLQHSxdeybF8QqbaYKNQ6j0obcg5BRhEjSmgI3NkhnhP/pub?start=false&loop=false&delayms=3000&slide=id.p1). 
+
+Such a platform would allow game theory incentives and smart contract based marketplaces for data trading to encourage adoption of the reference application. 
+
+Our team has years of research and development on medical data along with senior blockchain developers working with IT capable PhD accredited medical doctor(s). Data collection extraction and real time scaling is also part of Dr S Marvasti's and Dr Mahdi Ghandi's resume .
+Furthermore, Dr Marvasti has multiple years experience working with blockchain technologies including ZKP both for cross border fintech and for data analysis through AWS Neptune and AWS Aurora API based databases.
+This is a phased plan with part of the design to be implemented in this first phase, which each phase being usable .
+in summary benefits of blockchain for this data includes
+* Decentralised: Data ownership is not lost to any central authority and access is controlled through secure  NFT like smart contract
+* Incentive to participate: various mechanisms  could be developed as smart tokens or native token of the parachain linked to DOT.
+* Users:  publishers of health can receive some amount of token as rewards from any sponsor on the platform. 
+* Public: The api and parachain would be public so that any other apps such that deal with medical data can use this parachain. 
+* The detailed API and openAPI specifications for GRPC and or REST will be documented as part of this proposal.
+### Project Details
+
+Data models
+
+We propose to store data encrypted at client side using the Pod api implemented as a simple database on top of the off chain worker local storage. The data would be anonymous at the storage layer and
```

Comment:
This will become clearer once we develop the prototype demo application against the modified Substrate node. So yes the substrate node will be extended to have DB like functionality for application usage (not just the DB that most blockchain nodes have for node management and chain management). But something like SQLite DB built in to the off chain worker storage.